apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -471,6 +472,11 @@ defmodule OMG.Watcher.ExitProcessor.Core do
       invalid_exit_positions
       |> Enum.map(fn position -> ExitInfo.make_event_data(Event.InvalidExit, position, exits[position]) end)
 
+    # get exits which are invalid because of being spent in IFEs
+    invalid_exits_based_on_ifes_events =
+      get_invalid_exits_based_on_ifes(state)
+      |> Enum.map(fn {position, exit_info} -> ExitInfo.make_event_data(Event.InvalidExit, position, exit_info) end)
```

Comment:
why do you need a variable invalid_exits_based_on_ifes_events? 
and why it is so far away from its use?

Reply:
I moved close to use. I make do it without extra variable but I think in this way is more readable